tests/components/almond/test_config_flow.py

```diff
@@ -1,19 +1,31 @@
 """Test the Almond config flow."""
 import asyncio
 
+import pytest
+
 from homeassistant import config_entries, data_entry_flow, setup
 from homeassistant.components.almond import config_flow
 from homeassistant.components.almond.const import DOMAIN
 from homeassistant.const import CONF_CLIENT_ID, CONF_CLIENT_SECRET
 from homeassistant.helpers import config_entry_oauth2_flow
 
-from tests.async_mock import patch
+from tests.async_mock import Mock, patch
 from tests.common import MockConfigEntry
 
 CLIENT_ID_VALUE = "1234"
 CLIENT_SECRET_VALUE = "5678"
 
 
+@pytest.fixture
+def mock_current_request(hass):
```

Comment:
Should this fixture be part of the common conftest fixtures?

Reply:
Yeah, that might be a good idea, maybe even leverage the request mocks provided by aiohttp.

Reply:
Added in d7fc2dd